CIPs/CIP-23/CIP-23.md

```diff
@@ -0,0 +1,164 @@
+| `claim`        | Unique location where the DID string is stored. Should be a place where only the given ID could have posted the DID. | string           | 450 char  | optional | https://gist.github.com/marysmith/5c48debdb7 |
+| `attestations` | Attestations issued by third-party services which have verified the claim against the ID. | array of strings | 1000 char | optional | ceramic://bafy...123                         |
+
+**Deployment:** `<reference-schema-DocID>`
+
+```jsx
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "title": "SocialAccounts",
+  "items": {
+    "$ref": "#/definitions/Account"
+  },
+  "definitions": {
+    "Account": {
+      "type": "object",
+      "properties": {
+        "protocol": {
+          "type": "string",
+		      "maxLength": 50
+        },
+        "host": {
+          "type": "string",
+		      "maxLength": 150
+        },
+        "id": {
+          "type": "string",
+		      "maxLength": 450
+        },
+        "claim": {
+          "type": "string",
+		      "maxLength": 450
+        },
+        "attestations": {
+          "type": "array"
+          "items": {
+            "type": "string",
+		      	"maxLength": 1000
+          }
+        }
+      },
+      "required": [
+        "protocol",
+        "host",
+        "id"
+      ]
+    }
+  }
+}
+```
+
+### Tags
+
+In the config of the *definition* defined above we can see a `tags` property. We simply apply the tags in the config to the `tags` property in the *metadata* of the *reference* Tile.
+
+### Example
+
+An example Crypto Accounts *reference* document that includes two Ethereum accounts and one Bitcoin account.
+
+```js
+const profile = await ceramic.createDocument('tile', {
+  metadata: {
+    schema: "<reference-schema-DocID>"
+    tags: ["SocialAccounts"]
+  },
+  content: [{
+		protocol: "https",
+		host: "https://twitter.com",
+		id: "https://twitter.com/marysmith",
+    // ID of tweet containing the user's DID
+		claim: "https://twitter.com/marysmith/status/1274020265417076736",
+		attestations: ["eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."]
+	}, {
+  	protocol: "https"
+		host: "https://github.com"
+		id: "https://github.com/marysmith"
+    // ID of Gist containing the user's DID
+		claim: "https://gist.github.com/marysmith/5c48debdb7089b3c8f86cca31739572c"
+		attestations: ["eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."]
+	}]
+})
+```
+
+
+## Suggested Usage
+
+It is recomended to always include both the `claim` and the `attestation` property. The claim is usually a way to verify that the social account actually want's to be associated with the users DID. The attestation is a [Verifiable Credential](https://github.com/decentralized-identity/did-jwt-vc) from a third party that this is the case. In some scenarious it might be impossible to provide a publicly verifiable `claim`. In these cases just an `attestation` will have to suffice.
```

Comment:
I think we could make the last sentence more descriptive by saying that we can include the ID of a non-public post, but in these cases an attestation is required and there will be more trust in the attestation since it's not a publicly verifiable claim. (or something along those lines)